c/src/sbp.c

```diff
@@ -383,7 +389,7 @@ s8 sbp_process(sbp_state_t *s, s32 (*read)(u8 *buff, u32 n, void *context))
       if (s->crc == crc) {
 
         /* Message complete, process it. */
-        s8 ret = sbp_process_payload(s, s->sender_id, s->msg_type, s->msg_len,
+        s8 ret = (*sbp_process_payload)(s, s->sender_id, s->msg_type, s->msg_len,
```

Comment:
If you give this arg a different name then you shouldn't need to deference the function pointer here.